### ISSUE TITLE
Emits 4 forward slashes (`////`) for `/// /`

### ISSUE THREAD

**Issue by boozook** [rust-lang/rust-clippy#17392]
### Summary

`/// /` is not `////` and it's valid doc-comment,  
but `clippy` produces warning (by default) by lint `four_forward_slashes`, which is wrong.

### Lint Name

four_forward_slashes

### Reproducer

I tried this code:

```rust
/// / doc-comment,
/// // not "4 forward slashes"
/// too
mod __ {}
```

> doc of `__`:
> ```text
> / doc-comment
> // not “4 forward slashes”
> too
> ```
> __and it's totally correct.__

I saw this happen:

```
this item has comments with 4 forward slashes (`////`). These look like doc comments, but they aren't
for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#four_forward_slashes
`#[warn(clippy::four_forward_slashes)]` on by default
```

I expected to see this happen:

Nothing to warn.   
_Maybe only about invalid markdown because of it's doc-comment, but in spec doc-comment is just a text, so that's unlikely and better not._

### Version

```text
rustc 1.98.0-nightly (c397dae80 2026-07-02)
binary: rustc
commit-hash: c397dae808f70caebab1fc4e11b3edf7e59f58c7
commit-date: 2026-07-02
host: aarch64-apple-darwin
release: 1.98.0-nightly
LLVM version: 22.1.8
```

**Comment by boozook** [comment]
My mistake, sorry 🤦🏻‍♂️
